docs/_site/usage-modes.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
Should the `_site` directory be checked in or will github automatically build the HTML from the markdown?

Reply:
Good catch, looks like GH will do it for us

Reply:
Should this be on the `.gitignore` as well then?

Reply:
I think so, yeah